src/partisan_client_server_peer_service_manager.erl

```diff
@@ -210,15 +210,27 @@ handle_call({join, {Name, _, _}=Node},
 
 handle_call({send_message, Name, Message}, _From,
             #state{connections=Connections}=State) ->
-    Result = do_send_message(Name, Message, Connections),
-    {reply, Result, State};
+    %% find the node that goes by this name
+    case partisan_peer_service_connections:find(Name, Connections) of
+        {ok, Node} ->
+            Result = do_send_message(Node, Message, Connections),
+            {reply, Result, State};
+        not_found ->
+            {reply, not_found, State}
+    end;
 
 handle_call({forward_message, Name, ServerRef, Message}, _From,
             #state{connections=Connections}=State) ->
-    Result = do_send_message(Name,
-                             {forward_message, ServerRef, Message},
-                             Connections),
-    {reply, Result, State};
+    %% find the node that goes by this name
+    case partisan_peer_service_connections:find(Name, Connections) of
+        {ok, Node} ->
+            Result = do_send_message(Node,
+                                     {forward_message, ServerRef, Message},
+                                     Connections),
+            {reply, Result, State};
+        not_found ->
+            {reply, not_found, State}
```

Comment:
This is a new return value for the `not_found` case.  What happened previously if the connection wasn't found?